app-backend/api/src/main/scala/team/Routes.scala

```diff
@@ -0,0 +1,89 @@
+package com.azavea.rf.api.team
+
+import com.azavea.rf.common.{Authentication, CommonHandlers, UserErrorHandler}
+import com.azavea.rf.database.TeamDao
+import com.azavea.rf.database.filter.Filterables._
+import com.azavea.rf.datamodel._
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.model.StatusCodes
+import com.lonelyplanet.akka.http.extensions.PaginationDirectives
+import io.circe._
+import de.heikoseeberger.akkahttpcirce.ErrorAccumulatingCirceSupport._
+import java.util.UUID
+
+import cats.effect.IO
+
+import doobie.util.transactor.Transactor
+import cats.implicits._
+import doobie._
+import doobie.implicits._
+import doobie.Fragments.in
+import doobie.postgres._
+import doobie.postgres.implicits._
+
+
+/**
+  * Routes for Organizations
+  */
+trait TeamRoutes extends Authentication
+    with PaginationDirectives
+    with CommonHandlers
+    with UserErrorHandler {
+
+  val xa: Transactor[IO]
+
+  val teamRoutes: Route = handleExceptions(userExceptionHandler) {
+    pathEndOrSingleSlash {
+      get { listTeams } ~
```

Comment:
We've been wrapping our routes with Kamon traces which are missing here. We should discuss as team if we should continue this.